docs/Translation.md

```diff
@@ -0,0 +1,78 @@
+
+![Translation 2](./img/Translations/02_Translation.png)
+
+![Translation 3](./img/Translations/03_Translation_creation.png)
+
+![Translation 4](./img/Translations/04_Translation_dictionary.png)
+
+
+As seen in the image above, we have a dictionary of words that are translatable on BTCPay Server instance.
+
+Your dictionary is structured as (key: value), where:
+
+**Key:** The original English text or phrase found in your BTCPay instance.
+
+**Value:** The translated text you wish to display instead.
+
+For each English term, enter its equivalent in your chosen language in the corresponding text. Make sure to review the translations for accuracy and clarity.
+
+As an example lets start by translating "Add Role" into Yoruba. Since I created a dictionary for Yoruba language I would translate to Yoruba language.
+
+![Translation 5](./img/Translations/05_Translation_Add_Role_To_Yoruba.png)
+
+Replace the text and click the **Save** button. A confirmation message will appear indicating that your translations have been successfully saved. Now click on the **Select** button for the newly created dictionary to update the default dictionary on the system. 
+
+![Translation 6](./img/Translations/06_Translation_Saved_Dictionary.png)
+
+Now to test out our translation. You can navigate to **Roles** under **Server Settings**, and if we look at the button on the top right of the Roles view, we can see that the text has been successfully translated from "Add Roles" to "Fi ipa kun".
+
+![Translation 7](./img/Translations/07_Translation_Validation.png)
+
+We can search other part of the BTCPay application that has "Add Role" text, and we can also confirm that it has successfully been translated to our chosen language text. 
+
+Now that you have successfully translated one, go ahead and translate other texts in the dictionary. All instances of English text will be replaced with your translated text, and now you can enjoy your new localized experience.
+
+## Tips for Effective Translations
+
+- **Consistency is Key:** Ensure that similar terms are translated consistently throughout.
+- **Context Matters:** Keep context in mind when translating phrases to maintain meaning.
+- **Review Regularly:** Regularly update translations as new features are added.
+
+
+The translation feature on BTCPay Server allows you to create a personalized experience by managing content in your desired language. By following the steps highlighted above, you can easily set up and maintain translations, enhancing usability for yourself and your members.
+
+## Support
+
+Feel free to join our support channel over at [Mattermost chat](http://chat.btcpayserver.org/) if you need help or have any further questions.
+
+If experience a bug or you would like to report any missing translation string, please open an issue in our [Github repository](https://github.com/btcpayserver/btcpayserver/issues)
+
+
```

Comment:
Don't know these are just words, obvious chatgpt not really helpful so I'd remove them.
```suggestion
```